### PR TITLE
Updated PouchDB

### PR DESCRIPTION
**Pull request by tigerxy**
Updated for current version form 0.1 to 5.0.0.

### CODE REVIEW

**Comment by vvakame** [comment]
*pouchDB/pouch.d.ts*

to author(@MrBigDog2U). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

File: pouchDB/pouch.d.ts [hunk 1]
```diff
@@ -1,230 +1,134 @@
-// Type definitions for Pouch 0.1
+// Type definitions for PouchDB 5.0.0
 // Project: http://pouchdb.com
-// Definitions by: Bill Sears <https://github.com/MrBigDog2U/>
```
Comment (vvakame) [inline pouchDB/pouch.d.ts]: please keep original auther name.

**Comment by vvakame** [comment]
please check this document.
http://definitelytyped.org/guides/best-practices.html#ghost-modules

File: pouchDB/pouch.d.ts (hunk 1 above)
Comment (tigerxy) [inline pouchDB/pouch.d.ts]: I write the definition complete new. Should we store the old one in a seperately file?

**Comment by MrBigDog2U** [comment]
No.

As I've explained before, I no longer support this. Please remove my name.

File: pouchDB/pouch.d.ts (hunk 1 above)
Comment (vvakame) [inline pouchDB/pouch.d.ts]: 1st author is 1st user. I want to get review by 1st user (and you).

**Comment by vvakame** [comment]
@MrBigDog2U Where there is a mistake?

**Comment by tigerxy** [comment]
@vvakame please merge it.

File: pouchDB/pouch.d.ts [hunk 2]
```diff
@@ -1,230 +1,140 @@
...
+	}
+	interface PouchDBPromise {
+		then(callback: (result: Object) => void): PouchDBPromise
+		catch(callback: (err: Object) => void): PouchDBPromise
+	}
+}
+
+declare var PouchDB: PouchDB.PouchDBFactory
+
+declare module "PouchDB" {
```
Comment (polco) [inline pouchDB/pouch.d.ts]: module names are usually in lowercase. and I had to replace `export = PouchDB` by `export default PouchDB` to allow a default import ```import PouchDB from 'pouchdb';```

**Comment by mhegazy** [comment]
Can you refresh and address merge conflicts please.

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**